whistl/label_samples.py

```diff
@@ -280,7 +279,10 @@ def update_labels(labels=set()):
             sample_label = get_sample_label(sample, sample_data, labels, fields)
             if not sample_label:
                 break
-            labeled_samples[sample_label].append(sample)
+            if sample_label in labeled_samples:
+                labeled_samples[sample_label].append(sample)
+            else:
+                labeled_samples[sample_label] = [sample]
```

Comment:
What happens if a sample does not have a label?  As in there is a label for some samples and not others?  I'm not  too familiar with what the metadata looks like for refine.bio so perhaps this will never be the case,

Reply:
Since this script is designed as an aid for manually labeling samples, the loop that populates `sample_label` will never return without a label being assigned (see the `get_sample_label` function)